src/pybind/mgr/dashboard/frontend/src/app/ceph/block/rbd-configuration-form/rbd-configuration-form.component.html

```diff
@@ -5,7 +5,7 @@
     <h3 class="page-header">
       <span
         (click)="toggleSectionVisibility(section.class)"
-        class="collapsible">{{ section.heading }} <i [ngClass]="{'fa-plus-circle': !sectionVisibility[section.class], 'fa-minus-circle': sectionVisibility[section.class]}" class="fa" aria-hidden="true"></i></span>
+        class="collapsible">{{ section.heading }} <i [ngClass]="{ 'fa-plus-circle' : !sectionVisibility[section.class], 'fa-minus-circle': sectionVisibility[section.class]}" class="fa" aria-hidden="true"></i></span>
```

Comment:
```suggestion
        class="collapsible">{{ section.heading }} <i [ngClass]="{ icons.addCircle : !sectionVisibility[section.class], 'fa-minus-circle': sectionVisibility[section.class]}" class="fa" aria-hidden="true"></i></span>
```
Not sure if `fa-minus-circle` should be replaced with another icon or added to the list of icons...

Reply:
I might have missed it, thanks

Reply:
```suggestion
        class="collapsible">{{ section.heading }} <i [ngClass]="!sectionVisibility[section.class] ? icons.addCircle : icons.minusCircle" class="fa" aria-hidden="true"></i></span>
```